vidarr/wdl.py

```diff
@@ -174,7 +178,7 @@ def read_output(output: WDL.Decl):
             return output_metadata["vidarr_type"]
```

Comment:
This feature lets output_meta override wdl2vidarr's opinion about what type an output should be (https://github.com/oicr-gsi/vidarr-tools?tab=readme-ov-file#wdl2vidarr). Only in the 'else' case does your code appear to get called - if an output is _hardcoded_ to File using this feature, then 'vidarr_label' is provided, do you think it should get upgraded again to file-with-labels, or should wdl2vidarr honour the request and warn about the conflict?

Reply:
I think we should warn about the conflict because if a vidarr_type is specified, the workflow developer may not want the output vidarr type to be changed regardless of the existence of a label.